simplex/block_test.go

```diff
@@ -157,139 +163,87 @@ func TestVerify(t *testing.T) {
 	require.Equal(t, blockBytes, vBlockBytes, "block bytes should match after verification")
 }
 
-// TestVerifyPrevIsLatest tests that a block with a prev digest that is not found in the block tracker
-// successfully verifies the block if it is the latest accepted block.
```

Comment:
This test was no longer relevant. It seems like this test was written for a previous version of the code.